metricbeat/module/elasticsearch/elasticsearch.go

```diff
@@ -332,12 +332,6 @@ func MergeClusterSettings(clusterSettings common.MapStr) (common.MapStr, error)
 	return settings, nil
 }
 
-// IsCCRStatsAPIAvailable returns whether the CCR stats API is available in the given version
-// of Elasticsearch.
-func IsCCRStatsAPIAvailable(currentElasticsearchVersion string) (bool, error) {
```

Comment:
There is a test missing this function :slightly_smiling_face:

Reply:
Ugh, sorry about that. I shouldn't try to work on the PR while also rushing to make it to Elastic{ON} ;) Will fix tomorrow.

Reply:
Yeah, enjoy Elastic{ON} :smiley: